turbo/generators/config.ts

```diff
@@ -59,24 +91,30 @@ module.exports = function (plop) {
         name: 'name',
```

Comment:
Should this be `name: 'path'` instead?

Reply:
I think that makes sense yea, but didn't want to change any logic, just port to turbo generators.

Reply:
I guess I was confused because you added `validate` function that wasn't there before and that one specifically says `path`.